### ISSUE TITLE
Refresh search index for `odo.dev`

### ISSUE THREAD

**Issue by rm3l** [redhat-developer/odo#6194]
/area documentation
/kind bug

## What mistake did you find / what is missing in the documentation?
As reported by a user, some invalid links might pop up when doing some search on [odo.dev](https://odo.dev) and clicking on mentioned links.

[Screencast from 10-04-2022 05:00:19 PM.webm](https://user-images.githubusercontent.com/593208/193854973-b014b196-29fa-4052-81c7-7408129fd142.webm)

## What is the relevance of it?
Search might return invalid links.

**Comment by rm3l** [comment]
@cdrage Any idea how we could refresh the search index?

**Comment by rm3l** [comment]
Issue fixed.

Note for our future selves:
- #6203 made sure we were using the right application ID and search API Key
- the [crawler](https://crawler.algolia.com/admin/crawlers?sort=status&order=ASC&limit=20&appId=7RBQSTPIA4) was paused by Algolia a few months ago; since we had performed a lot of documentation changes, it would have resulted in a high number of lost records in the previous index. As a [safety measure](https://www.algolia.com/doc/tools/crawler/apis/configuration/safety-checks/), the crawler stopped refreshing the index. Let's make sure we are notified when that situation occurs. The threshold (default is 10% of total records) seems to be configurable though.
- the [index](https://www.algolia.com/apps/7RBQSTPIA4/explorer/browse/odo?page=1&searchMode=search) was too old and stopped refreshing.

To fix this, as recommended in the Algolia status page for the crawler, we needed to delete the old index, and start a new crawl (which recreated a new index).

Closing this as completed, as the search is now working as expected.
Now let's keep an eye on the next crawler runs and index age..

/close
